src/components/TransactionDetails/TransactionDetailsList.jsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable camelcase */
+import React from 'react';
+
+const TransactionDetailsList = ({ operations }) => (
+  <>
+    <ul className="transactions_list">
+      {operations.map(opr => {
+        const { output_value, value, hash } = opr;
+        const [address] = opr.addresses;
+        return (
```

Comment:
could you add an empty line before return to improve the code readability please

Reply:
Fixed everywhere